### ISSUE TITLE
Path to rethinkdb_web_assets relative to current directory

### ISSUE THREAD

**Issue by t3hmrman** [rethinkdb/rethinkdb#3309]
tldr; If you try to `rethinkdb serve` and the `rethinkdb` binary is linked to somewhere else (with ln -s, for example), binary can't find rethinkdb_web_assets folder

Scenario:
1. Fresh install of rethinkdb to ~/path/to/rethinkdb
2. `cd ~/bin`
3. `ln -s ~/path/to/rethinkdb`
4. Add ~/bin to PATH
5. `which rethinkdb` returns the binary you would expect (at the right location)
6. `rethinkdb create` in a folder somewhere
7. `rethinkdb serve` and you get an error saying it can't find rethinkdb_web_assets (which is in ~/path/to/rethinkdb/build/release)
   7a.`~/path/to/rethinkdb/build/release/rethinkdb serve` does work

**Workaround**
Use alias
